src/wave_controller/wave.py

```diff
@@ -31,8 +29,9 @@ class RootWave(MDBoxLayout):
     current_harmonic_index = 0
 
     def __init__(self, **kwargs: typing.Any):
-        super(RootWave, self).__init__(**kwargs)
+        super().__init__(**kwargs)
         self.max_samples_per_harmonic = int(self.harmonic_samples.max)
+        z = 2
```

Comment:
<sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>C0103: Variable name "z" doesn't conform to snake_case naming style (invalid-name)